Classes/Http/Middleware/Preview.php

```diff
@@ -34,16 +35,15 @@ class Preview implements MiddlewareInterface
 {
     public const REQUEST_ATTRIBUTE = 'tx_authorized_preview_config';
 
-    protected Context $context;
-
-    public function __construct(Context $context)
-    {
-        $this->context = $context;
-    }
+    public function __construct(protected readonly Context $context) {}
```

Comment:
sorry, this is PHP 8.x feature, but the current code supports also PHP 7.4

Reply:
not anymore we dont 😀